src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -137,13 +137,13 @@ public static object GetUninitializedObjectWithFormatterServices(Type type)
         private static SerializationContext serializationContext;
 
         private static SerializationContext CurrentSerializationContext
-            => serializationContext ?? (serializationContext = new SerializationContext());
+            => serializationContext ?? (serializationContext = new SerializationContext(null));
         
         [ThreadStatic]
         private static DeserializationContext deserializationContext;
 
         private static DeserializationContext CurrentDeserializationContext
-            => deserializationContext ?? (deserializationContext = new DeserializationContext());
+            => deserializationContext ?? (deserializationContext = new DeserializationContext(null));
```

Comment:
these look error prone... why are they used? Isn't it better to return null if unset?